teacher.cpp

```diff
@@ -43,7 +45,7 @@ Teacher::Teacher(const std::string& teacher_conf,
 void Teacher::set_py_task_paths() {
     //// list all the relative paths of the python tasks here
     std::vector<std::string> relative_python_paths = {
-        "./games/xworld/tasks",
```

Comment:
both tasks are needed. You shouldn't just delete xworld. Otherwise people won't be able to run xworld2d tasks.

Reply:
then we need to rename all task and map files for 3d?